ocs_ci/ocs/scale_lib.py

```diff
@@ -108,6 +110,8 @@ def create_multi_pvc_pod(
             io_runtime (seconds): Runtime in Seconds to continue IO
             pvc_size (int): Size of PVC to be created
             max_pvc_size (int): The max size of the pvc
+            remove_security_context_section (bool): If True, remove the security context section from the Yaml file.
+                False, otherwise. Default value is True.
```

Comment:
You can remove all the code changes in this PR related to security_context.
Also rebase the PR if not done recently to pull the latest changes which has the perf yaml changes